app/services/hulu_mapping_service.rb

```diff
@@ -0,0 +1,83 @@
+# Service to sync Hulu API with Kitsu
+class HuluMappingService
+  HULU_URL = 'https://distribution.hulu.com/api/v1/'.freeze
+
+  attr_reader :since
+
+  # Initialize service with optional datetime
+  def initialize(since = nil)
+    @since = since
+  end
+
+  # Grabs all available series from hulu
+  def get_series_for_episodes(limit)
+    offset = 0
+    series_res = Typhoeus.get(
+      "#{HULU_URL}/series?guid=#{guid}&primary_category=animation&offset=#{offset}&limit=#{limit}"
+    )
+    return unless series_res.success?
+    JSON.parse(series_res.body)['results']
+  end
+
+  # Start of syncing series and episodes on hulu with kitsu anime
+  def sync_series_and_episodes
+    limit = 10_000
+    hulu_anime = get_series_for_episodes(limit)
+    return if hulu_anime.nil?
+    hulu_anime.each do |item|
+      item = item.deep_symbolize_keys
+      series_id = item[:id]
+      title = item[:name].titleize
+      mapping_object = {
+        title: title
+      }
+      kitsu_anime = Mapping.guess('Anime', mapping_object)
```

Comment:
`Mapping.guess` should generally be preceded by an attempt to find an existing mapping with `Mapping.lookup`

Also you can probably just do `Mapping.guess('Anime', title: title)` instead of building `mapping_object`